legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/domain/api/entity/EntityAccessContext.java

```diff
@@ -26,5 +26,7 @@
 
     List<Entity> getEntities(Predicate<String> entityPathPredicate, Predicate<String> classifierPathPredicate, Predicate<? super Map<String, ?>> entityContentPredicate);
 
+    List<Entity> getInvalidEntities(Predicate<String> entityPathPredicate, Predicate<String> classifierPathPredicate, Predicate<? super Map<String, ?>> entityContentPredicate);
```

Comment:
This cannot return Entity, since what this will return is not a valid entity. There needs to be a new type for this (maybe call it InvalidEntity), which gives information on why this is not a valid entity and provides some kind of information that might help in fixing the entity. For the former, I would recommend an error message and/or stack trace. For the latter, I would recommend a string containing the actual source code.

It's also not clear whether classifierPathPredicate or entityContentPredicate really make sense here. Since we can't assume that the entity can be deserialized, we can't be sure we'll be able to test those predicates.

Finally, there should be an API to get a single invalid entity by its path.